pkgs/development/python-modules/astroid/default.nix

```diff
@@ -23,7 +56,9 @@ buildPythonPackage rec {
     ++ lib.optional (pythonOlder "3.5") typing
     ++ lib.optional (!isPyPy) typed-ast;
```

Comment:
I think we do not need typed-ast for 3.9 https://github.com/PyCQA/astroid/blob/astroid-2.4.2/astroid/__pkginfo__.py#L32

Reply:
In fact we only need it < 3.8.